src/test/groovy/de/iteratec/osm/result/WptResultValidationSpec.groovy

```diff
@@ -0,0 +1,51 @@
+package de.iteratec.osm.result
+
+import de.iteratec.osm.measurement.environment.wptserver.WptResultXml
+import groovy.util.slurpersupport.GPathResult
+import spock.lang.Specification
+
+/**
+ * @author nkuhn
+ */
+class WptResultValidationSpec extends Specification {
+
+    void "WPT Result validation"() {
+        setup: "given a result node from wpt"
+        GPathResult xmlResult = new XmlSlurper().parseText("""
+            <firstView>
+                <result>${wptStatus}</result>
+                <TTFB>${ttfb}</TTFB>
+                <render>${startRender}</render>
+                <visualComplete>${visComplete}</visualComplete>
+                <SpeedIndex>${speedIndex}</SpeedIndex>
+                <docTime>${docComplete}</docTime>
+                <fullyLoaded>${fullyLoadedTime}</fullyLoaded>
+            </firstView>
+            """)
+        WptResultXml resultXml = new WptResultXml(xmlResult, 10, 60000)
+
+        expect: "it gets validated to #valid"
+        resultXml.isValidTestStep(xmlResult) == valid
+
+        where:
+        wptStatus   | ttfb  | startRender   | visComplete   | speedIndex    | docComplete   | fullyLoadedTime || valid
```

Comment:
Is it possible to get -1 values? As far as I remember, there where cases where the default value was -1.

And I'm not quite sure, if we also should check wpt error codes with 0/-1 on all metrics, like for status 0. The current code will stop on these codes, but changes in the future may brake this behavior. Any thoughts on this?